src/components/list/partials/list-card.vue

```diff
@@ -31,6 +31,7 @@
 			</BaseButton>
 
 			<div class="title">{{ list.title }}</div>
+			<div class="description">{{ list.description }}</div>
```

Comment:
I don't think showing the description is a good idea. If there's more than 2-3 lines (which can be very fast) there won't be enough space to show it. Combine that with a long list name and it gets worse.